iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/bicep/PropertyImplTest.java

```diff
@@ -41,15 +42,28 @@
 import static org.assertj.core.api.Assertions.catchThrowableOfType;
 import static org.sonar.iac.common.testing.IacTestUtils.code;
 
-class PropertyImplTest {
+class PropertyImplTest extends BicepTreeModelTest {
 
-  BicepParser parser = BicepParser.create(BicepLexicalGrammar.PROPERTY);
+  @Test
+  void shouldParseProperty() {
+    Assertions.assertThat(BicepLexicalGrammar.PROPERTY)
+      .matches("key:value")
+      .matches("'key':value")
+      .matches("key: value")
+      .matches("key :value")
+      .matches("key : value")
```

Comment:
There are some tests from peter's PR that I would like to see here aswell. 
Maybe substitute `key` to these examples, after you rebase (after Peter merged his PR)
```
'a${123}b${456}c'
'a${123}${456}c'
```